proto/zitadel/auth.proto

```diff
@@ -552,6 +627,61 @@ message ListMyUserSessionsResponse {
     repeated zitadel.user.v1.Session result = 1;
 }
 
+message ListMyMetaDataRequest {
+    zitadel.v1.ListQuery query = 1;
+    repeated zitadel.metadata.v1.MetaDataQuery queries = 2;
+}
+
+message ListMyMetaDataResponse {
+    zitadel.v1.ListDetails details = 1;
+    repeated zitadel.metadata.v1.MetaData result = 2;
+}
+
+message GetMyMetaDataRequest {
+    string key = 1 [(validate.rules).string = {min_len: 1}];
+}
+
+message GetMyMetaDataResponse {
+    zitadel.metadata.v1.MetaData meta_data = 1;
+}
+
+message SetMyMetaDataRequest {
+    string key = 1 [(validate.rules).string = {min_len: 1, max_len: 200}];
+    string value = 2 [(validate.rules).string = {min_len: 1}];
```

Comment:
I discussed this with livio and we said, that you can store everything in a string. but i can also change it.

Reply:
At the moment we don't have a maximum size, to see what kind of values will be stored. its realy difficult to set e max size if you don't know for what kind of data it will be used

Reply:
> I discussed this with livio and we said, that you can store everything in a string. but i can also change it.

I see your point, but for me a string is a (UTF-8) text and should not contain data of another type, if multiple datatypes should be stored in a field i recommend using bytes

Reply:
> At the moment we don't have a maximum size, to see what kind of values will be stored. its realy difficult to set e max size if you don't know for what kind of data it will be used

Good point, but i would still set a max size, it can be a high one as 5mb or higher. I just want to minimise the chance that some chaos monkey can attack our API  with large requests

Reply:
i set a max size. at the moment its not really high. we will see if we need a higher one